Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env groovy
```

Comment:
Why do we need to create a PR if it's already in master? I think I may be missing something